deploy/charts/rook-ceph/values.yaml

```diff
@@ -624,7 +624,7 @@ discover:
   #       memory: 128Mi
 
 # -- Custom label to identify node hostname. If not set `kubernetes.io/hostname` will be used
-customHostnameLabel: 
+customHostnameLabel:
```

Comment:
This is an auto-fix from my editor. Is this space here for a specific purpose? @arttor

Reply:
I think the yaml spec ignores trailing whitespace.

Reply:
correct, whitespace is not needed here.